events/smart-descriptions.json

```diff
@@ -1,4 +1,294 @@
+        {
+          "field": "event.category",
+          "value": "traffic"
+        },
+        {
+          "field": "action.type",
+          "value": "start"
+        },
+        {
+          "field": "network.transport",
+          "value": "udp"
+        }
+      ],
+      "relationships": [
+        {
+          "source": "source.ip",
+          "target": "destination.ip",
+          "type": "session started"
+        }
+      ]
+    },
+    {
+      "value": "{action.name}{event.category}:{action.type} - Session denied between {source.ip}:{source.port} and {destination.ip}:{destination.port}",
+      "conditions": [
+        {
+          "field": "event.category",
+          "value": "traffic"
+        },
+        {
+          "field": "action.type",
+          "value": "deny"
+        },
+        {
+          "field": "network.transport",
+          "value": "tcp"
+        }
+      ],
+      "relationships": [
+        {
+          "source": "source.ip",
+          "target": "destination.ip",
+          "type": "session denied"
+        }
+      ]
+    },
+    {
+      "value": "{action.name}{event.category}:{action.type} - Session dropped between {source.ip}:{source.port} and {destination.ip}:{destination.port}",
+      "conditions": [
+        {
+          "field": "event.category",
+          "value": "traffic"
+        },
+        {
+          "field": "action.type",
+          "value": "drop"
+        },
+        {
+          "field": "network.transport",
+          "value": "tcp"
+        }
+      ],
+      "relationships": [
+        {
+          "source": "source.ip",
+          "target": "destination.ip",
+          "type": "session dropped"
+        }
+      ]
+    },
+    {
+      "value": "{action.name}{event.category}:{action.type} - Session ended between {source.ip}:{source.port} and {destination.ip}:{destination.port}",
+      "conditions": [
+        {
+          "field": "event.category",
+          "value": "traffic"
+        },
+        {
+          "field": "action.type",
+          "value": "end"
+        },
+        {
+          "field": "network.transport",
+          "value": "tcp"
+        }
+      ],
+      "relationships": [
+        {
+          "source": "source.ip",
+          "target": "destination.ip",
+          "type": "session ended"
+        }
+      ]
+    },
+    {
+      "value": "{action.name}{event.category}:{action.type} - Session started between {source.ip}:{source.port} and {destination.ip}:{destination.port}",
+      "conditions": [
+        {
+          "field": "event.category",
+          "value": "traffic"
+        },
+        {
+          "field": "action.type",
+          "value": "start"
+        },
+        {
+          "field": "network.transport",
+          "value": "tcp"
+        }
+      ],
+      "relationships": [
+        {
+          "source": "source.ip",
+          "target": "destination.ip",
+          "type": "session started"
+        }
+      ]
+    },
```

Comment:
```suggestion
```

I don't think we need to use the `network.transport` distinction since there is no difference in generated descriptions. Or did I miss something ?